src/webview_provider/insightsPanel.ts

```diff
@@ -118,6 +121,137 @@ export class InsightsPanel extends AltimateWebviewProvider {
     }
   }
 
+  private async fetchProjectIntegrations(syncRequestId: string | undefined) {
+    try {
+      console.log("Fetching project integrations");
+      const response = await this.altimateRequest.fetchProjectIntegrations();
+      if (!response) {
```

Comment:
No response means bad credentials? There must be other error conditions as well.

Reply:
Yes, removing it as `fetch` function already handles the same thing.